oemof/db/feedin_pg.py

```diff
@@ -19,8 +18,28 @@ def __init__(self):
         ''
         pass
 
-    def create_fixed_source(self, conn, **kwargs):
-        ''
+    def aggregate_cap_val(self, conn, **kwargs):
+        '''
+        Returns the normalised feedin profile and installed capacity for
+        a given region.
+
+        Parameters
+        ----------
+        region : Region instance
+        region.geom : shapely.geometry object
+            Geo-spatial data with information for location/region-shape. The
+            geometry can be a polygon/multi-polygon or a point.
+
+        Returns
+        -------
+        feedin_df : pandas dataframe
+            Dataframe containing the normalised feedin profile of the given
```

Comment:
I do not see where you normalise the feedin profile. I thought there would be something like 
```python
for source_type in feedin_df.keys():
    feedin_df[source_type] = feedin_df[source_type] / cap[source_type]
```
Or do I misunderstand something?